src/components/product/ProductSummary/product-summary.scss

```diff
@@ -0,0 +1,78 @@
+.product-summary[data-store-card] {
+  position: relative;
+  display: flex;
+  flex-direction: column;
+  max-width: 14rem;
+  height: 100%;
+  padding: var(--space-1);
+  background-color: var(--bg-neutral-lightest);
+
+  &:hover {
+    border-radius: var(--border-radius-default);
```

Comment:
We can move the border-radius to the `.product-summary[data-store-card]` so it won't mess up the transition

Reply:
Sureee!